orga/tests/acceptance/campaign-list-test.js

```diff
@@ -32,7 +32,7 @@ module('Acceptance | Campaign List', function(hooks) {
 
     // then
     assert.equal(currentURL(), '/campagnes/liste');
-    assert.dom('.page__title').hasText('Campagnes');
+    assert.dom('.page__title').hasText('Créez votre première campagne');
```

Comment:
Pour le coup, le nom du test ne décrit pas cet attention

Reply:
je suis pour un test à part : 'it should display creation button when there is no campaign found'

Reply:
A séparer en deux